ocelot/urls.py

```diff
@@ -46,6 +50,9 @@
     path('teampage', teampage, name='teampage-view'),
     path('updates', updates, name='updates-view'),
     path('welcome', welcome, name='welcome-view'),
+    # evaluation app
+    path('compare/<sub_a_id>/<sub_b_id>', compare, name='compare-view'),
```

Comment:
@snukky: use `'submission/<sub_a_id>/<sub_b_id>'` for comparison? No need for a specific `compare` view name?

Reply:
That's a good idea! Changed as suggested.